### ISSUE TITLE
bug(MatPaginator): Screenreader (JAWS) reads disabled controls twice on Chromium based browsers

### ISSUE THREAD

**Issue by MrMightyNighty** [angular/components#26967]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

As the title mentions, JAWS reads deactivated controls twice. This issue only appears on Chromium based browsers (Chrome, Edge). On Firefox JAWS behaves correctly and reads the deactivated button only once. 

I also tested this with NVDA and NVDA works perfectly fine on all browsers but our company uses JAWS for its users.

### Reproduction

Steps to reproduce:
1. Start JAWS
2. Visit https://material.angular.io/components/paginator/overview and use arrow keys (up and down) to navigate to any example paginator
3. Cycle through controls with arrow keys
4. Deactivated buttons (e.g. "previous button" when it is the first page) can be navigated twice.

### Expected Behavior

JAWS should read deactivated controls once

### Actual Behavior

JAWS reads deactivated controls twice 

### Environment

- Angular: 15.2.8
- CDK/Material: 15.2.8
- Browser(s): Chrome, Edge, Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10

**Comment by amysorto** [comment]
I am unable to navigate to the paginator and buttons using arrow keys (can only do so using tab). Can you provide a video with what you are seeing?

**Comment by zarend** [comment]
I can reproduce using by tabbing to an activated "Next page" button, then navigating back with "Say Prior Word" command.

Assigning to myself, I have a fix for this.

Say Prior Word | INSERT+LEFT ARROW | CAPS LOCK+J
-- | -- | --
Say Next Word | INSERT+RIGHT ARROW | CAPS LOCK+L

**Comment by MrMightyNighty** [comment]
> I am unable to navigate to the paginator and buttons using arrow keys (can only do so using tab). Can you provide a video with what you are seeing?

Sadly I'm not able to provide a video since I'm using a dev license and we aren't allowed to record anything due to our company policies...

At least @zarend was able to reproduce it (not with arrow keys but at least something :) )
